pkg/component/component.go

```diff
@@ -526,19 +526,85 @@ func ValidateComponentCreateRequest(client *occlient.Client, componentSettings c
 //	appName: Name of application of which the component is a part
 //	componentName: Name of the component which is being patched with config
 //	componentConfig: Component configuration
+//  cmpExist: true if components exists in the cluster
```

Comment:
indentation

Reply:
fixed

Reply:
still there :-(

Reply:
not sure what is the problem, but my IDE shows that it is fixed :(
tried with vscode, goland and vim

Reply:
somehow fixed :smile_cat: